src/test/resources/docker/SimpleGradle_8_2.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gradle:8.2.1-jdk11
+
+# Do not change SRC_DIR, value is expected by tests
+ENV SRC_DIR=/opt/project/src
+
+ENV JAVA_TOOL_OPTIONS="-Dhttps.protocols=TLSv1.2"
+
+
+# Install git
+RUN apt-get update && apt-get install -y git
+
+# Set up the test project
+RUN mkdir -p ${SRC_DIR}
+
+
+RUN git clone --depth 1 https://github.com/atiqzaman-dsi/simple-gradle.git ${SRC_DIR}
```

Comment:
What happens when the employee leaves the company? We shouldn't be using personal projects.

Reply:
This is just a sample project publicly available on github.

Reply:
In this case, https://github.com/atiqzaman-dsi/simple-gradle looks like Atiq's personal project and the test might fail if that project or his account gets deleted. (I also agree and would feel comfortable if we have a repo in blackducksoftware to place these sample projects there so we own them).

Reply:
Also think it'd be valuable to clone a specific release branch of these sample projects with `git clone <project> -b <release-tag>` instead of default main. This would avoid tests to fail if we want to validate BOM components in these tests in the future (as we do in some other integration tests).

Reply:
But it's tied to an individual account. Once someone leaves the company, he/she can make it a private project or just delete the whole project. In that case, the build starts failing. 
I know it's a very simple project but I feel it should be under an organization and not tied to any individual. Not a good practice.

Reply:
Also, if the individual becomes a bad actor or the account is hacked then one can introduce a malicious code through `gradle build` in our CI server which can affect a lot of different products inside Synopsys VPN.